### ISSUE TITLE
ContextMenuOverlay does not work as shown in example

### ISSUE THREAD

**Issue by Zarainia** [gskinnerTeam/flutter-context-menu#4]
The example shows:

```dart
return ContextMenuOverlay(
  child: MaterialApp(...)
);
```

but when actually creating such a layout with this code: 

```dart
import 'package:flutter/material.dart';
import 'package:context_menus/context_menus.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return ContextMenuOverlay(child: MaterialApp(
      title: 'Flutter Demo',
      home: Scaffold(
        appBar: AppBar(
          title: Text("Test"),
        ),
        body: Column(children: [],),
      ),
    ));
  }
}
```

I get this error:

```
======== Exception caught by widgets library =======================================================
The following assertion was thrown building _InheritedContextMenu:
No MediaQuery widget ancestor found.

Scaffold widgets require a MediaQuery widget ancestor.
The specific widget that could not find a MediaQuery ancestor was: Scaffold
  dirty
  state: ScaffoldState#48e24(lifecycle state: initialized, tickers: tracking 2 tickers)
The ownership chain for the affected widget is: "Scaffold ← _InheritedContextMenu ← LayoutBuilder ← ContextMenuOverlay ← MyApp ← [root]"

No MediaQuery ancestor could be found starting from the context that was passed to MediaQuery.of(). This can happen because you have not added a WidgetsApp, CupertinoApp, or MaterialApp widget (those widgets introduce a MediaQuery), or it can happen if the context you use comes from a widget above those widgets.

The relevant error-causing widget was: 
  _InheritedContextMenu file:///C:/Users/Zarai/AppData/Local/Pub/Cache/hosted/pub.dartlang.org/context_menus-0.1.0+5/lib/src/context_menu_overlay.dart:73:16
When the exception was thrown, this was the stack: 
#0      debugCheckHasMediaQuery.<anonymous closure> (package:flutter/src/widgets/debug.dart:219:7)
#1      debugCheckHasMediaQuery (package:flutter/src/widgets/debug.dart:234:4)
#2      MediaQuery.of (package:flutter/src/widgets/media_query.dart:820:12)
#3      ScaffoldState.didChangeDependencies (package:flutter/src/material/scaffold.dart:2831:50)
#4      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4732:11)
...
====================================================================================================
```

which doesn't happen when the `ContextMenuOverlay` is inside `MaterialApp`.

**Comment by sxd1140** [comment]
Same here.
```dart
    return ContextMenuOverlay(
      child: MaterialApp.router(
```

**Comment by PureTryOut** [comment]
This makes it impossible to use when using different constructors like `MaterialApp.router()`...

**Comment by esDotDev** [comment]
Fixed in v1.1.1

**Comment by PureTryOut** [comment]
Awesome! Any chance you could tag your releases on git? It's a great way to track what's exactly in each release for people outside your project.
